src/Sylius/Behat/spec/Context/Domain/ProductContextSpec.php

```diff
@@ -44,90 +53,150 @@ function it_implements_context_interface()
         $this->shouldImplement(Context::class);
     }
 
-    function it_deletes_a_product_variant_from_the_repository(
-        ProductVariantRepositoryInterface $productVariantRepository,
-        ProductVariantInterface $productVariant
-    ) {
-        $productVariantRepository->remove($productVariant)->shouldBeCalled();
+    function it_deletes_a_product_variant_from_the_repository($variantRepository, ProductVariantInterface $productVariant)
+    {
+        $variantRepository->remove($productVariant)->shouldBeCalled();
 
         $this->iDeleteTheVariantOfProduct($productVariant);
     }
 
-    function it_tries_to_delete_a_product_variant_that_should_not_be_delted_from_the_repository(
-        SharedStorageInterface $sharedStorage,
```

Comment:
It does, it make specs more readable. At least for me. But it is arguable if this benefit is better then autocomplete